tests/js/HappinessLintBearTest.py

```diff
@@ -17,3 +21,18 @@
 HappinessLintBearTest = verify_local_bear(HappinessLintBear,
                                           valid_files=(good_file,),
                                           invalid_files=(bad_file,))
+
+
+@generate_skip_decorator(HappinessLintBear)
+class HappinessLintBearConfigTest(LocalBearTestHelper):
+
+    VALUE_ERR_RE = ('Set "use_spaces = False" for HappinessLintBear'
+                    'to avoid conflicts with other bears')
+
+    def test_validate_use_spaces(self):
+        section = Section('name')
+        section.append(Setting('use_spaces', True))
+        bear = HappinessLintBear(section, Queue())
+
+        with self.assertRaisesRegex(AssertionError, self.VALUE_ERR_RE):
+            self.check_validity(bear, [], good_file)
```

Comment:
this is kinda misusing `self.check_validity`^^

Reply:
okk so what changes should i make?

Reply:
just run the bear without an assertion